### ISSUE TITLE
Adding a note to the NGP VAN apply_activist_code function

### ISSUE THREAD

**Issue by vischalla** [move-coop/parsons#445]
Can we add a note in the Parsons docs on the NGP VAN appy_activist_code function that using it will also apply a canvass status to the person? A member raised that this caused an issue in their canvassing counts. It wasn’t clear why the previously loaded activist codes were impacting canvass results without looking at the underlying VAN API docs.

Source docs: https://developers.ngpvan.com/van-api#activist-codes

**Comment by jburchard** [comment]
What was the canvass mode that was listed?

**Comment by bzupnick** [comment]
Is this issue closed from that merged PR? Or should we be adding something in the docs of this behavior?

**Comment by shaunagm** [comment]
No, the issue is still open - there's nothing in the docs about this.
